docs/reference/reference-training/experiment-config-reference.rst

```diff
@@ -1105,7 +1105,7 @@ experiment.
    For multi-GPU training, whether to average the training metrics across GPUs instead of only using
    metrics from the chief GPU. This impacts the metrics shown in the Determined UI and TensorBoard,
    but does not impact the outcome of training or hyperparameter search. This option is currently
-   only supported in PyTorch. Defaults to ``false``.
+   only supported in PyTorch. Defaults to ``true``.
```

Comment:
it's actually supported in PyTorchTrial and TFKerasTrial.

Also, now that we have Core API, referring to PyTorchTrial as "PyTorch" isn't really specific enough anymore.

Reply:
It's also marked at supported for DeepSpeedTrials, but possibly not accurately.

Reply:
I changed this to "PytorchTrial and TFKerasTrial instances" for now. Left DeepSpeed out of it, provisionally.  There are also other instances of bare "PyTorch" that could be replaced, but I left them for a larger docs clean-up PR.